src/styles/post-template.less

```diff
@@ -251,6 +252,10 @@ body {
             border-color: @wtfColor;
         }
 
+        &.team-content-platform {
```

Comment:
Looks like you need to copy this definition at around line 480 as well.

Reply:
Good catch! It looks like `wtf` and `mobile` are also missing entries in that part of the file. @aasmundeldhuset Either you can copy those two while you're at it, or I can do it in a separate commit after this one is done.

Reply:
Thanks - fixed!